.cicd/pipeline.yml

```diff
@@ -292,7 +292,7 @@ steps:
     agents:
       queue: "automation-eks-eos-tester-fleet"
     timeout: ${TIMEOUT:-10}
-    skip: ${SKIP_UBUNTU_18}${SKIP_UNIT_TESTS}
+    skip: ${SKIP_UBUNTU_18}${SKIP_INTEGRATION_TESTS}
```

Comment:
Why is integration test to be skipped?

Reply:
This flag gives you the option to skip the tests by setting an ENV var on a specific build. This is really useful when debugging a different part of the CI pipeline.

Reply:
Thanks for your explanation! Didn't realize it is a flag.